jsonnet/components/metrics-server.libsonnet

```diff
@@ -0,0 +1,377 @@
+function(params) {
+  local cfg = params,
+
+  serviceAccount: {
+    apiVersion: 'v1',
+    kind: 'ServiceAccount',
+    metadata: {
+      labels: {
+        'app.kubernetes.io/name': 'metrics-server',
+        'app.kubernetes.io/component': 'metrics-server',
+      } + cfg.commonLabels,
+      name: 'metrics-server',
+      namespace: cfg.namespace,
+    },
+  },
+  clusterRoleAggregatedMetricsReader: {
```

Comment:
can we move it to the cluster-monitoring-operator component to avoid duplication with prometheus-adapter?

Reply:
I created separate PR https://github.com/openshift/cluster-monitoring-operator/pull/2163 since that includes more changes too